src/test/util.h

```diff
@@ -295,8 +295,15 @@ inline static void* allocate_guard(size_t size, char value) {
  */
 inline static void verify_guard(__attribute__((unused)) size_t size, void* p) {
   char* cp = (char*)p;
+  int tmp_errno = errno;
   test_assert(
       memcmp(cp - sizeof(GUARD_VALUE), &GUARD_VALUE, sizeof(GUARD_VALUE)) == 0);
+  static int verify_guard_checksum;
```

Comment:
Why is this `static`?

Reply:
This is from a state when I was not yet sure how to get it to the rr register check. I thought first about doing a printf of it before program exit, but that would make it hard to find out when things start to go wrong.
Calling the invalid syscall is now done in each verify_guard, so no need for static.